llvm/test/Other/lit.local.cfg

```diff
@@ -0,0 +1 @@
+config.suffixes = ['', '.ll']
```

Comment:
This is because in LIT `%s` expands to the source file path with periods escaped like so: `.../dump-before-after-filenames\\.`, which won't match the output of `find`. 

I also considered running `ls` on each file, but I think the output of filecheck on find is a little easier to read when it fails.